packages/composables/src/helpers/urlIsAbsolute.ts

```diff
@@ -0,0 +1,3 @@
+export default function getUrlPrefix(url: string) {
```

Comment:
Is it good practice to name the default function different than the function where I use it?